src/Compilers/CSharp/Test/Emit/CodeGen/FixedSizeBufferTests.cs

```diff
@@ -1134,5 +1134,87 @@ .maxstack  1
   IL_001f:  ret
 }");
         }
+
+        [Fact, WorkItem(39632, "https://github.com/dotnet/roslyn/issues/39632")]
+        public void CanUseIndexInitialiserWithFixedSizeBuffer()
+        {
+            var source = @"
+using System;
+
+unsafe struct S
+{
+    public fixed double V[3];
+
+    static void Main()
+    {
+        var b = new S();
+        b.V[0] = 0;
```

Comment:
I'm confused, wasn't this meant to test an object initializer?

Reply:
oops